admin/app/components/certification/certification-details-answer.js

```diff
@@ -31,6 +31,14 @@ export default class CertificationDetailsAnswer extends Component {
     return this.hasJuryResult ? 'answer-result jury' : 'answer-result';
   }
 
+  get linkToChallengePreviewInPixApp() {
+    return `https://app.pix.fr/challenges/${this.args.answer.challengeId}/preview`;
+  }
+
+  get linkToChallengeInfoInPixEditor() {
+    return `https://editor.pix.fr/#/competence/${this.args.competenceId}/prototypes/${this.args.answer.challengeId}`;
```

Comment:
Pour info `https://editor.pix.fr/#/challenge/${this.args.answer.challengeId}` fonctionne aussi

Reply:
Ah ! je vais mettre ça alors, c'est plus joli et ça ne m'oblige pas à traîner le competenceId de plus haut. Mercix !